src/InverseSolve/models.jl

```diff
@@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
 	return f(m),sdiag(df(m))
 end
 
+function dummyMod(m)
```

Comment:
thanks for the explanation. I think what we should use is 

```
DiagonalScaling(1.0)
```
This is treated like a sparse identity (of any size) also, for example, when adding it to another square sparse matrix. How about changing the name of your method from ```dummyMod``` to something more meaningful, for example, ```idMod```?